m4/nx-macros.m4

```diff
@@ -0,0 +1,314 @@
+# nx-macros.m4.  Generated from xorg-macros.m4.in xorgversion.m4 by configure.
```

Comment:
> Do we really have to duplicate X.Org code? Can't we depend upon and use util-macros directly for the few functions that we need? I understand that doing that might be more difficult (for now - at some point we'll have to use util-macros within the X.Org code base anyway) and that copying makes the NX components more independent.

Of course, we can use util-marcos directly. Or rather, we could. However, that let's our builds fail on all older platforms where our customers and friends may need nx-libs to build.